scripts/irods/test/test_misc.py

```diff
@@ -317,3 +319,32 @@ def test_server_correctly_cleans_up_proc_files__issue_6231(self):
         # Show that the proc directory will eventually become empty.
         lib.delayAssert(lambda: len(os.listdir(paths.proc_directory())) == 0)
 
+    def test_iinit_does_not_crash_from_long_home_directory__issue_5411(self):
```

Comment:
Test currently does not provide enough information upon encountering an error.

Log errors like `assert_icommand` does, and if possible, don't quit upon the first failed assertion, so more information can be gathered.